edx_dl/edx_dl.py

```diff
@@ -536,15 +538,17 @@ def parse_courses(args, available_courses):
         exit(ExitCode.OK)
 
     if len(args.course_urls) == 0:
-        logging.error('You must pass the URL of at least one course, check the correct url with --list-courses')
+        logging.error('You must pass the URL of at least one course, ' + 
+                      'check the correct url with --list-courses')
```

Comment:
This is not OK. Breaking a long message (that has no way of being shorter) simply creates an artificial break that does not improve the code, like the change before. Please, drop it.

Reply:
The long message split into multiple lines has been restored. Pylint won't stop noticing the long line. May need some other way of addressing this.

Reply:
@vaidyasm, that's not a problem. We can add annotations so that pylint will skip those problems.